### PR TITLE
Fix occupancy does not calculate correctly

### PR DESCRIPTION
**Pull request by KKyang**

### CODE REVIEW

**Comment by jichangjichang** [comment]
Please describe the issue/root cause/ solution in the PR's description

File: tensilelite/Tensile/KernelWriterAssembly.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def getVgprOccupancy(self, numThreads, vgprs, doubleVgpr=False):
     maxOccupancy = self.consts.maxOccupancy//multiplier
 
     vgprAllocateAligned = 4    if not doubleVgpr else 8
-    totalVgprs = self.consts.maxVgprs if not doubleVgpr else self.consts.maxVgprs*2
```
Comment (jichangjichang) [inline tensilelite/Tensile/KernelWriterAssembly.py:77]: But vgprs could be 512, why did you remove the 2*maxVgrps?

**Comment by KKyang** [comment]
Close due to this is not the root cause of the issue
